src/helpers/client/safari-client.ts

```diff
@@ -48,16 +48,25 @@ let keepAliveTimeout: ReturnType<typeof setTimeout>;
  * Handles communication with the Safari extension using JSON-RPC over custom events.
  */
 export class SafariClient implements Client {
-  private safariExtensionExecutors: Map<string, PromiseExecutor>;
+  private statusInterval = 200;
+  private keepAliveInterval = 1000;
+  private promiseExecutors: Map<string, PromiseExecutor>;
+
+  private lastSentPing: number = null;
```

Comment:
Should type be declared as number|null same for lastReceivedPong

Reply:
Done!